frontend/src/pages/download/Download.jsx

```diff
@@ -1,13 +1,28 @@
 import React, { useState, useEffect, useMemo } from "react";
 import "./style.scss";
-import { Row, Col, Typography, Table, Button, Space, notification } from "antd";
+import {
+  Row,
+  Col,
+  Typography,
+  Table,
+  Button,
+  Space,
+  notification,
+  Select,
+} from "antd";
 import { api, store } from "../../lib";
 import { uiText } from "../../static";
 import { useNotification } from "../../util";
 import moment from "moment";
 
 const { Title } = Typography;
 
+const status = [
```

Comment:
Hi @navins94 , 
A little update for status static text, 
Can you move this into `ui-text.js` file, please?

Reply:
And when I click Request button, that return error 500, 
I think an error on the serializer, and please create a new test case to show saved data on download list and request to download saved data.